ci-operator/step-registry/ovn/conf/ovn-conf-commands.sh

```diff
@@ -3,8 +3,6 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
-
 touch "${SHARED_DIR}/install-config.yaml"
 /tmp/yq w -i "${SHARED_DIR}/install-config.yaml" 'networking.networkType' OVNKubernetes
```

Comment:
this line doesn't work now. hmmm.... 

from [this job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/38149/rehearse-38149-pull-ci-openshift-cluster-samples-operator-release-4.13-e2e-aws-upgrade/1646338949754392576):

`Error: 1:1: invalid input text "w"
`

Reply:
yeah, looks like this older version 3.3.0 allows that w argument, but the newer on in cli-yq (v4.31.2) does not

old:
```
 /tmp/yq --version
yq version 3.3.0
❯ /tmp/yq w -i /dev/null 'jamo' 'jamo'
12:06:04 safelyRenameFile [ERRO] Failed copying from /tmp/temp632143218 to /dev/null
12:06:04 safelyRenameFile [ERRO] sync /dev/null: invalid argument
```

new:
```
[root@749bdd98a1be /]# /tmp/yq --version
yq (https://github.com/mikefarah/yq/) version v4.31.2
[root@749bdd98a1be /]# /tmp/yq w -i /dev/null 'jamo' 'jamo'
Error: 1:1: invalid input text "w"
[root@749bdd98a1be /]# 
```

Reply:
did some testing and I think you can update the command to be:

`/tmp/yq -i '.networking.networkType = "OVNKubernetes"' "${SHARED_DIR}/install-config.yaml"`

Reply:
Ah, yeah I should have tested locally with the image instead of trying for pj-rehearse.  Fixed.